### ISSUE TITLE
`/ccrackrng`, when it fails, may print `Failed to crack player seed` multiple times at seemingly-random intervals

### ISSUE THREAD

**Issue by James103** [Earthcomputer/clientcommands#387]
As of Client Commands 2.6.5 for Minecraft 1.18.2, when `/ccrackrng` fails, it may output `Failed to crack player seed` multiple times instead of only once. When it does print multiple times, it does so at seemingly-random intervals and it may keep going until RNG manipulation is stopped or restarted via sprinting, changing dimensions, respawning, or running `/ccrackrng` again (among others).

<details>

<summary> Example output in log when this occurs </summary>

At 17:21:45, a player typed `/ccrackrng`, then went AFK for ~30 s. The following was what was logged on their client during that time.
```
[17:21:45] [Render thread/INFO]: [CHAT] Failed to crack player seed
[17:21:45] [Render thread/INFO]: [CHAT] Help: RNG manipulation doesn't work on some modded servers, in particular Paper.
[17:21:45] [Render thread/INFO]: [CHAT] Failed to crack player seed
[17:21:45] [Render thread/INFO]: [CHAT] Help: RNG manipulation doesn't work on some modded servers, in particular Paper.
[17:21:45] [Render thread/INFO]: [CHAT] Failed to crack player seed
[17:21:45] [Render thread/INFO]: [CHAT] Help: RNG manipulation doesn't work on some modded servers, in particular Paper.
[17:21:46] [Render thread/INFO]: [CHAT] Failed to crack player seed
[17:21:46] [Render thread/INFO]: [CHAT] Help: RNG manipulation doesn't work on some modded servers, in particular Paper.
[17:21:46] [Render thread/INFO]: [CHAT] Failed to crack player seed
[17:21:46] [Render thread/INFO]: [CHAT] Help: RNG manipulation doesn't work on some modded servers, in particular Paper.
[17:21:47] [Render thread/INFO]: [CHAT] Failed to crack player seed
[17:21:47] [Render thread/INFO]: [CHAT] Help: RNG manipulation doesn't work on some modded servers, in particular Paper.
[17:21:50] [Render thread/INFO]: [CHAT] Failed to crack player seed
[17:21:50] [Render thread/INFO]: [CHAT] Help: RNG manipulation doesn't work on some modded servers, in particular Paper.
[17:21:51] [Render thread/INFO]: [CHAT] Failed to crack player seed
[17:21:51] [Render thread/INFO]: [CHAT] Help: RNG manipulation doesn't work on some modded servers, in particular Paper.
[17:21:52] [Render thread/INFO]: [CHAT] Failed to crack player seed
[17:21:52] [Render thread/INFO]: [CHAT] Help: RNG manipulation doesn't work on some modded servers, in particular Paper.
[17:21:52] [Render thread/INFO]: [CHAT] Failed to crack player seed
[17:21:52] [Render thread/INFO]: [CHAT] Help: RNG manipulation doesn't work on some modded servers, in particular Paper.
[17:21:54] [Render thread/INFO]: [CHAT] Failed to crack player seed
[17:21:54] [Render thread/INFO]: [CHAT] Help: RNG manipulation doesn't work on some modded servers, in particular Paper.
[17:21:55] [Render thread/INFO]: [CHAT] Failed to crack player seed
[17:21:55] [Render thread/INFO]: [CHAT] Help: RNG manipulation doesn't work on some modded servers, in particular Paper.
[17:21:55] [Render thread/INFO]: [CHAT] Failed to crack player seed
[17:21:55] [Render thread/INFO]: [CHAT] Help: RNG manipulation doesn't work on some modded servers, in particular Paper.
[17:21:56] [Render thread/INFO]: [CHAT] Failed to crack player seed
[17:21:56] [Render thread/INFO]: [CHAT] Help: RNG manipulation doesn't work on some modded servers, in particular Paper.
[17:22:02] [Render thread/INFO]: [CHAT] Failed to crack player seed
[17:22:02] [Render thread/INFO]: [CHAT] Help: RNG manipulation doesn't work on some modded servers, in particular Paper.
[17:22:03] [Render thread/INFO]: [CHAT] Failed to crack player seed
[17:22:03] [Render thread/INFO]: [CHAT] Help: RNG manipulation doesn't work on some modded servers, in particular Paper.
[17:22:08] [Render thread/INFO]: [CHAT] Failed to crack player seed
[17:22:08] [Render thread/INFO]: [CHAT] Help: RNG manipulation doesn't work on some modded servers, in particular Paper.
[17:22:09] [Render thread/INFO]: [CHAT] Failed to crack player seed
[17:22:09] [Render thread/INFO]: [CHAT] Help: RNG manipulation doesn't work on some modded servers, in particular Paper.
[17:22:13] [Render thread/INFO]: [CHAT] Failed to crack player seed
[17:22:13] [Render thread/INFO]: [CHAT] Help: RNG manipulation doesn't work on some modded servers, in particular Paper.
```

</details>

<details>

<summary> Mod list </summary>

```
  [✔️] advancementinfo-1.18-fabric0.43.1-1.3
  [✔️] carpet-extra-1.18.2-pre2-1.4.64
  [✔️] clientcommands-2.6.5
  [✔️] fabric-api-0.47.8+1.18.2
  [✔️] fabric-carpet-1.18.2-1.4.66+v220228
  [✔️] ferritecore-4.2.0-fabric
  [✔️] itemscroller-fabric-1.18.2-0.16.0
  [✔️] lazydfu-0.1.2
  [✔️] litematica-fabric-1.18.2-0.11.0
  [✔️] lithium-fabric-mc1.18.2-0.7.8-SNAPSHOT
  [✔️] malilib-fabric-1.18.2-0.12.0
  [✔️] minihud-fabric-1.18.2-0.22.0
  [✔️] modmenu-3.1.0
  [✔️] notenoughcrashes-4.1.4+1.18.2-fabric
  [✔️] sodium-fabric-mc1.18.2-0.4.1+build.15
  [✔️] starlight-1.0.2+fabric.89b8d9f
  [✔️] tweakeroo-fabric-1.18.2-0.13.1
  [❌] Xaeros_Minimap_22.3.0_Fabric_1.18.2.jar (disabled)
  [❌] Xaeros_Minimap_22.3.1.1_Fabric_1.18.2.jar (disabled)
  [✔️] Xaeros_Minimap_22.4.0_Fabric_1.18.2
  [❌] XaerosWorldMap_1.20.3.1_Fabric_1.18.2.jar (disabled)
  [❌] XaerosWorldMap_1.20.4.1_Fabric_1.18.2.jar (disabled)
  [❌] XaerosWorldMap_1.20.5_Fabric_1.18.2.jar (disabled)
  [✔️] XaerosWorldMap_1.20.6_Fabric_1.18.2
```

</details>

**Comment by Earthcomputer** [comment]
Does this still occur on the latest version?

**Comment by TeraV8** [comment]
This still happens on 2.7.13

**Comment by xpple** [comment]
If the issue somehow still occurs, please open a new issue.
